monitoring/mock_uss/geoawareness/parsers/ed269/__init__.py

```diff
@@ -0,0 +1,151 @@
+import json
```

Comment:
I think it's fine to leave this in for now, but I think this would be a strong candidate for moving to [uas_standards](https://github.com/interuss/uas_standards) when time allows.

Reply:
Let's keep it here for few more PRs and I will move it there when mature.